src/templates/Home/state/types.ts

```diff
@@ -0,0 +1,17 @@
+export interface Todo {
+  id: string;
+  text: string;
+}
+
+export type ColumnId =
```

Comment:
For now this code is fine. But if we are thinking on making the user able to add columns, having this fixed names wont work. maybe we can use a { [key]: string: {...data} } approach.
the key would be the column title

Reply:
I agree, but I wanted to make 100% sure that the state outside of the columns was not refreshing. Later on we can make this a normal string.